### PR TITLE
Add child logger & remove method names from log messages

### PR DESCRIPTION
**Pull request by ian-shim**
## Why are these changes needed?
Refactoring loggers across codebase. 
- Adds child loggers with `component` attribute for better filtering
- Removes method names from log messages as they're already logged as part of `source` attribute

## Checks

- [x] I've made sure the lint is passing in this PR.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: common/geth/client.go [hunk 1]
```diff
@@ -265,13 +266,13 @@ func (c *EthClient) waitMined(ctx context.Context, txs []*types.Transaction) (*t
...
 					if receipt.BlockNumber.Uint64()+uint64(c.numConfirmations) > chainTip {
-						c.Logger.Debug("EnsureTransactionEvaled: transaction has been mined but don't have enough confirmations at current chain tip", "txnBlockNumber", receipt.BlockNumber.Uint64(), "numConfirmations", c.numConfirmations, "chainTip", chainTip)
+						c.Logger.Debug("transaction has been mined but don't have enough confirmations at current chain tip", "txnBlockNumber", receipt.BlockNumber.Uint64(), "numConfirmations", c.numConfirmations, "chainTip", chainTip)
 						break
 					} else {
 						return receipt, nil
 					}
 				} else {
-					c.Logger.Debug("EnsureTransactionEvaled: failed to get chain tip while waiting for transaction to mine", "err", err)
+					c.Logger.Debug("failed to get chain tip while waiting for transaction to mine", "err", err)
```
Comment (pschork) [inline common/geth/client.go:275]: nit: `failed to query block height`

File: disperser/batcher/txn_manager.go [hunk 2]
```diff
@@ -261,7 +261,7 @@ func (t *txnManager) monitorTransaction(ctx context.Context, req *TxnRequest) (*
 			}
 			txID, err := t.wallet.SendTransaction(ctx, newTx)
 			if err != nil {
-				t.logger.Error("failed to send txn", "component", "TxnManager", "method", "monitorTransaction", "tag", req.Tag, "txn", req.Tx.Hash().Hex(), "attempt", retryFromFailure, "maxRetry", maxSpeedUpRetry, "err", err)
+				t.logger.Warn("failed to send txn", "component", "TxnManager", "method", "monitorTransaction", "tag", req.Tag, "txn", req.Tx.Hash().Hex(), "attempt", retryFromFailure, "maxRetry", maxSpeedUpRetry, "err", err)
```
Comment (pschork) [inline disperser/batcher/txn_manager.go:264]: Maybe log `Warn("failed to send txn - retrying"` when retrying and `Error("failed to send txn - retries exhaused"` when we give up.

File: common/geth/client.go [hunk 3]
```diff
@@ -265,13 +266,13 @@ func (c *EthClient) waitMined(ctx context.Context, txs []*types.Transaction) (*t
 				chainTip, err := c.BlockNumber(ctx)
 				if err == nil {
 					if receipt.BlockNumber.Uint64()+uint64(c.numConfirmations) > chainTip {
-						c.Logger.Debug("EnsureTransactionEvaled: transaction has been mined but don't have enough confirmations at current chain tip", "txnBlockNumber", receipt.BlockNumber.Uint64(), "numConfirmations", c.numConfirmations, "chainTip", chainTip)
+						c.Logger.Debug("transaction has been mined but don't have enough confirmations at current chain tip", "txnBlockNumber", receipt.BlockNumber.Uint64(), "numConfirmations", c.numConfirmations, "chainTip", chainTip)
```
Comment (pschork) [inline common/geth/client.go:269]: nit: `don't -> doesn't` and `tip -> head`

**Review by pschork — APPROVED** [review]
Feel free to ignore the nits. When I see `tip` I think `gas tip` (ie priority fee), but maybe thats just me ¯\_(ツ)_/¯

File: retriever/metrics.go [hunk 4]
```diff
@@ -44,7 +44,7 @@ func NewMetrics(httpPort string, logger logging.Logger) *Metrics {
 			},
 		),
 		httpPort: httpPort,
-		logger:   logger,
+		logger:   logger.With("component", "retriever.Metrics"),
```
Comment (dmanc) [inline retriever/metrics.go:47]: nit: Should we standarize on camel case? So this should be "RetrieverMetrics"?

File: operators/churner/server.go [hunk 5]
```diff
@@ -39,7 +39,7 @@ func NewServer(
 		churner:                     churner,
 		latestExpiry:                int64(0),
 		lastRequestTimeByOperatorID: make(map[core.OperatorID]time.Time),
-		logger:                      logger,
+		logger:                      logger.With("component", "churner.Server"),
```
Comment (dmanc) [inline operators/churner/server.go:42]: Same comment for everything that has a "." in the name

**Review by dmanc — APPROVED** [review]

File: common/geth/client.go [hunk 6]
```diff
@@ -41,10 +41,11 @@ var _ common.EthClient = (*EthClient)(nil)
 // NewClient creates a new Ethereum client.
 // If PrivateKeyString in the config is empty, the client will not be able to send transactions, and it will use the senderAddress to create transactions.
 // If PrivateKeyString in the config is not empty, the client will be able to send transactions, and the senderAddress is ignored.
-func NewClient(config EthClientConfig, senderAddress gethcommon.Address, rpcIndex int, logger logging.Logger) (*EthClient, error) {
+func NewClient(config EthClientConfig, senderAddress gethcommon.Address, rpcIndex int, _logger logging.Logger) (*EthClient, error) {
```
Comment (jianoaix) [inline common/geth/client.go:44]: why adding a new naming convention with underscore prefix? May keep is simple and consistent with others

File: retriever/metrics.go (hunk 4 above)
Comment (ian-shim) [reply retriever/metrics.go:47]: I standardized it to the struct name and prefixed by the `<package_name>.` when the struct name is vague. 
I guess it's cleaner to just camel case it

**Pull request merged**